### PR TITLE
Best Practices > Anti-patterns: Added 'feature-coupled' step definition example for JS.

### LINKED ISSUE #113: [PR #114] Add javascript example "Feature-coupled step definitions (anti-pattern)"
There's one example missing for javascript in `anti-patterns.md`.
Java and Ruby examples for same are provided.
Missing example is marked "TODO" in the file.

### PR DESCRIPTION
**Pull request by xtrasimplicity**
Added a 'feature-coupled' step definition to provide an example of the anti-pattern's implementation, in Javascript, to resolve: https://github.com/cucumber/docs.cucumber.io/issues/113

---
This change is [Reviewable](https://reviewable.io/reviews/cucumber/docs.cucumber.io/114)

### CODE REVIEW

**Comment by mlvandijk** [comment]
Hi @charlierudolph - Would you please review this javascript example?

**Comment by mlvandijk** [comment]
this bit moved to anti-patterns.md

File: content/best-practices.md [hunk 1]
```diff
@@ -124,7 +124,14 @@ The{{% text "java" %}}`edit_work_experience_steps.java`{{% /text %}}
...
 
 ```javascript
-TODO: See Java or Ruby example
+    var { Given } = require('cucumber');
+
+    Given(/^I have a CV and I'm on the edit description page$/, function (callback) {
+      this.employee = new Employee('Sally');
+      this.employee.createCV();
+
+      callback();
```
Comment (charlierudolph) [inline content/best-practices.md]: You can remove the callback parameter to make it a synchronous step
```js
Given(/^I have a CV and I'm on the edit description page$/, function () {
  this.employee = new Employee('Sally');
  this.employee.createCV();
});
```

Comment (mlvandijk) [reply content/best-practices.md]: Thnx @charlierudolph !

**Comment by mlvandijk** [comment]
@xtrasimplicity If you don't mind, would you process @charlierudolph's comment and also, move this to the `antipatterns.md` (that change has been merged to master). Thnx for your help!

**Comment by xtrasimplicity** [comment]
All done! :)

**Pull request merged**